includes/Template/Woocommerce/Checkout/Woo_Checkout_Helper.php

```diff
@@ -412,22 +417,25 @@ public static function checkout_order_review_default($settings) {
 							<div class="table-col-1 product-thum-name">
 								<div class="product-thumbnail">
 									<?php
-									$thumbnail = apply_filters( 'woocommerce_cart_item_thumbnail', $_product->get_image(), $cart_item, $cart_item_key );
-									echo $thumbnail; // PHPCS: XSS ok.
```

Comment:
Did we remove this thumbnail?
Need to print this.

Reply:
No. Instead to use variable we print directly.  see next line`echo apply_filters( 'woocommerce_cart_item_thumbnail', $_product->get_image(), $cart_item, $cart_item_key );` 
![image](https://github.com/user-attachments/assets/f3516ba9-2aaf-462e-9c7c-7af9b019e931)